### PR TITLE
feat: support zipkin and console trace exporter

### PR DESCRIPTION
**Pull request by fatelei**
ref: https://github.com/cectc/dbpack/issues/<issueID>

### Ⅰ. Describe what this PR did

support new trace exporter
- zipkin
- console

### Ⅳ. Describe how to verify it

image

### Ⅴ. Special notes for reviews

break change:

update trace config

### CODE REVIEW

**Review by dk-lockdown — APPROVED** [review]
LGTM

File: pkg/config/config.go [hunk 1]
```diff
@@ -234,8 +235,9 @@ var _configuration = new(Configuration)
 func Load(path string) (*Configuration, error) {
 	configPath, _ := filepath.Abs(path)
 	log.Infof("load config from :  %s", configPath)
-	content, err := ioutil.ReadFile(configPath)
+	content, err := os.ReadFile(configPath)
 	if err != nil {
+		println("aaaa")
```
Comment (bohehe) [inline pkg/config/config.go]: Is this a testing code?

File: pkg/driver/client.go [hunk 2]
```diff
@@ -861,23 +861,23 @@ func (conn *BackendConnection) Ping(ctx context.Context) (err error) {
...
 //
-//       This is because closing a TCP socket on the server side sends
-//       a FIN to the client (telling the client the server is done
-//       writing), but on most platforms doesn't send a RST.  So the
-//       client has no idea it can't write. So it succeeds writing Content, which
-//       *then* triggers the server to send a RST back, received a bit
-//       later. By then, the client has already started waiting for
-//       the response, and will just return a CRServerLost(2013).
-//       So CRServerGone(2006) will almost never be seen with TCP.
+//		    This is because closing a TCP socket on the server side sends
```
Comment (bohehe) [inline pkg/driver/client.go:870]: Too much space here, is there a purpose for that?

**Review by bohehe — APPROVED** [review]

File: pkg/driver/client.go (hunk 2 above)
Comment (fatelei) [reply pkg/driver/client.go:870]: code format

**Pull request merged**